.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,62 @@
+name: Bug report
+description: Report an issue you encountered when playing OpenRCT2.
+labels: 
+- bug
+
+body:
+- type: markdown
+  attributes:
+    value: Before filing this bug, [please check if it hasn't already been reported](https://github.com/OpenRCT2/OpenRCT2/issues?q=is%3Aissue+is%3Aopen+label%3Abug). If it has, please add useful information to the existing issue instead.
```

Comment:
Can we change the filter to include both open + closed issues? Because the suggestion might have been something that we already discarded.